### PR TITLE
Improve tests

### PR DESCRIPTION
**Pull request by roback**
Made some small improvements to the tests.

### CODE REVIEW

File: spec/client_spec.rb [hunk 1]
```diff
@@ -1,44 +1,36 @@
...
   describe ".new" do
-    context 'with API key as arguments' do
-      it { should be_a Client }
-    end
+    context "with API key as argument" do
+      subject { described_class.new(valid_api_key) }
 
-    it "BASE_URL should be parsable" do
-      expect(URI(Client::BASE_URL).to_s).to eq(Client::BASE_URL)
+      it { should be_a Twingly::Search::Client }
```
Comment (walro) [inline spec/client_spec.rb]: noticed we changed to `described_class` in another test, do it here too?

(although I think we should use some other more appropriate matcher)

Comment (roback) [inline spec/client_spec.rb]: I just thought it looks weird checking that `described_class.new` is a `described_class`. It's easier to understand if its explicit, but the "correct way" would probably be to use `described_class` here too.

Comment (walro) [inline spec/client_spec.rb]: Alright

**Comment by walro** [comment]
I think this was an improvement, merge?

**Comment by roback** [comment]
Yes, I'll merge.

**Pull request merged**